### PR TITLE
Resize filter update

### PR DESCRIPTION
**Pull request by aptomalars**
Comments on this update?

Controversial points:
- Disabled auto escaping
- ~~Brittle encoding~~

### CODE REVIEW

File: src/filters/resize.js [hunk 1]
```diff
@@ -14,25 +14,23 @@ module.exports = resize;
...
-
-		url = url[0];
 		options = Object.assign({}, options);
 
 		// __keywords is added by nunjucks
 		delete options.__keywords;
 
-		const resize = [];
+		// remove all instances of t[resize] parameters in URL before adding only new values
+		url = url.replace(/(&|\?)t\[resize\]\[\w*\]=(<|>|\^|!|%|@|\d)*/g, '');
```
Comment (mipli) [inline src/filters/resize.js]: Are we avoiding using any kind of extra modules here? Might be easier to read these things if the `url` module was used instead.

Comment (aptomalars) [reply src/filters/resize.js]: No, not by choice. I was going for the simplest solution, but feature creep struck.

Comment (mipli) [reply src/filters/resize.js]: After some testing seems like the `url` module doesn't really work with our way of storing arrays in the query string. So regex might be the best solution then.

**Comment by peterudo** [comment]
It looks good, but I'm not a fan of all the regexp.

**Comment by aptomalars** [comment]
Is there an alternative that doesn't include parsing the URL? (As we only want to decode parts of the URL and not everything, and that we afterwards want to re-encode those parts, but again, not all of the URL)

**Comment by peterudo** [comment]
I'm not sure yet. I'll have a brief look at it.

**Comment by pahen** [comment]
We're using https://www.npmjs.com/package/qs in other projects for a lot of URL related things. Might be worth looking up ..

**Comment by peterudo** [comment]
I rewrote it now @aptomalars.

**Comment by aptomalars** [comment]
Looks good.

**Pull request merged**